mne_qt_browser/tests/test_pg_specific.py

```diff
@@ -81,3 +83,281 @@ def test_annotations_interactions(raw_orig, pg_backend):
+    assert pg_backend._get_n_figs() == 2
+    assert fig.mne.fig_settings != None
+    fig._fake_click_on_toolbar_action('Settings', wait_after=500)
+    assert pg_backend._get_n_figs() == 1
+    assert fig.mne.fig_settings == None
+
+    fig._fake_click_on_toolbar_action('Settings', wait_after=500)
+    assert fig.mne.fig_settings != None
+    downsampling_control = fig.mne.fig_settings.downsampling_box
+    assert downsampling_control.value() == fig.mne.downsampling
+
+    downsampling_control.setValue(2)
+    QTest.qWait(100)
+    assert downsampling_control.value() == 2
+    assert downsampling_control.value() == fig.mne.downsampling
+
+    downsampling_control.setValue(3)
+    QTest.qWait(100)
+    assert downsampling_control.value() == 3
+    assert downsampling_control.value() == fig.mne.downsampling
+
+    QTest.qWait(100)
+    downsampling_method_control = fig.mne.fig_settings.ds_method_cmbx
+    assert fig.mne.ds_method == downsampling_method_control.currentText()
+
+    downsampling_method_control.setCurrentText('mean')
+    QTest.qWait(100)
+    assert downsampling_method_control.currentText() == 'mean'
+    assert fig.mne.ds_method == 'mean'
+
+    downsampling_method_control.setCurrentText('subsample')
+    QTest.qWait(100)
+    assert downsampling_method_control.currentText() == 'subsample'
+    assert fig.mne.ds_method == 'subsample'
+
+    downsampling_method_control.setCurrentText('peak')
+    QTest.qWait(100)
+    assert downsampling_method_control.currentText() == 'peak'
+    assert fig.mne.ds_method == 'peak'
+
+    downsampling_method_control.setCurrentText('invalid_method_name')
+    QTest.qWait(100)
+    assert downsampling_method_control.currentText() != 'invalid_method_name'
+
+    sensitivity_control = fig.mne.fig_settings.scroll_sensitivity_slider
+    assert fig.mne.scroll_sensitivity == sensitivity_control.value()
+
+    sensitivity_control.setValue(100)
+    QTest.qWait(100)
+    assert sensitivity_control.value() == 100
+    assert fig.mne.scroll_sensitivity == 100
+
+    QTest.qWait(100)
+    sensitivity_values = list(range(sensitivity_control.minimum(), sensitivity_control.maximum() + 1, 40))
+    if sensitivity_values[-1] != sensitivity_control.maximum():
+        sensitivity_values.append(sensitivity_control.maximum())
+    sensitivity_test = [False] * len(sensitivity_values)
+    for i in range(len(sensitivity_values)):
+        sensitivity_control.setValue(sensitivity_values[i])
+        QTest.qWait(50)
+        sensitivity_test[i] = fig.mne.scroll_sensitivity == sensitivity_control.value()
+    assert all(sensitivity_test)
+
+    sensitivity_test = [False] * len(sensitivity_values)
+    for i in reversed(range(len(sensitivity_values))):
+        sensitivity_control.setValue(sensitivity_values[i])
+        QTest.qWait(50)
+        sensitivity_test[i] = fig.mne.scroll_sensitivity == sensitivity_control.value()
+    assert all(sensitivity_test)
+
```

Comment:
you should run flake8 on your code

Reply:
done: 3aefc9d989feb151580cc639dc0b33acd70a3284